packages/@aws-cdk/aws-appmesh/test/integ.mesh.expected.json

```diff
@@ -1409,6 +1544,126 @@
       }
     },
+    "meshgateway1gateway1routehttp2225001508": {
+      "Type": "AWS::AppMesh::GatewayRoute",
+      "Properties": {
+        "MeshName": {
+          "Fn::GetAtt": [
+            "meshACDFE68E",
+            "MeshName"
+          ]
+        },
+        "Spec": {
+          "Http2Route": {
+            "Action": {
+              "Rewrite": {
+                "Path": {
+                  "Exact": "/rewrittenpath"
+                }
+              },
+              "Target": {
+                "VirtualService": {
+                  "VirtualServiceName": {
+                    "Fn::GetAtt": [
+                      "service6D174F83",
+                      "VirtualServiceName"
+                    ]
+                  }
+                }
+              }
+            },
+            "Match": {
+              "Path": {
+                "Exact": "/exact"
+              }
+            }
+          }
+        },
+        "VirtualGatewayName": {
+          "Fn::GetAtt": [
+            "meshgateway1B02387E8",
+            "VirtualGatewayName"
+          ]
+        },
+        "GatewayRouteName": "meshstackmeshgateway1gateway1routehttp22BD49AE9D"
+      }
+    },
+    "meshgateway1gateway1routehttp2376EB99D6": {
+      "Type": "AWS::AppMesh::GatewayRoute",
+      "Properties": {
+        "MeshName": {
+          "Fn::GetAtt": [
+            "meshACDFE68E",
+            "MeshName"
+          ]
+        },
+        "Spec": {
+          "Http2Route": {
+            "Action": {
+              "Target": {
+                "VirtualService": {
+                  "VirtualServiceName": {
+                    "Fn::GetAtt": [
+                      "service6D174F83",
+                      "VirtualServiceName"
+                    ]
+                  }
+                }
+              }
+            },
+            "Match": {
+              "Path": {
+                "Regex": "regex"
+              }
+            }
+          }
+        },
+        "VirtualGatewayName": {
+          "Fn::GetAtt": [
+            "meshgateway1B02387E8",
+            "VirtualGatewayName"
+          ]
+        },
+        "GatewayRouteName": "meshstackmeshgateway1gateway1routehttp23E44F5774"
+      }
+    },
+    "meshgateway1gateway1routehttp245A894992": {
+      "Type": "AWS::AppMesh::GatewayRoute",
+      "Properties": {
+        "MeshName": {
+          "Fn::GetAtt": [
+            "meshACDFE68E",
+            "MeshName"
+          ]
+        },
+        "Spec": {
+          "Http2Route": {
+            "Action": {
+              "Target": {
+                "VirtualService": {
+                  "VirtualServiceName": {
+                    "Fn::GetAtt": [
+                      "service6D174F83",
+                      "VirtualServiceName"
+                    ]
+                  }
+                }
+              }
+            },
+            "Match": {
+              "Prefix": "path"
+            }
```

Comment:
> @Seiya6329 I see you also removed validation for path prefixes starting with `/`. Is that correct? Are you really planning to lift that requirement in the backend?

I added this integ test to verify that `prefix` does not need to start with `/`.